src/styles/framework.scss

```diff
@@ -200,32 +200,33 @@ span > a,
     &:not(.button):not(.no-link-decoration) {
         position: relative;
         text-decoration: none;
-        display: inline-block;
-        padding: 0 0px;
         @media (prefers-reduced-motion: no-preference) {
-            transition: color ease 0.3s;
+            transition: all ease 0.3s;
         }
 
-        &::after {
-            content: '';
-            position: absolute;
-            z-index: -1;
-            width: 100%;
-            height: 2px;
-            left: 0;
-            bottom: 0;
-            background-color: $accent-color;
-            @media (prefers-reduced-motion: no-preference) {
-                transition: all ease 0.3s;
+        // Very loosely inspired by: https://codepen.io/aaroniker/pen/pojaBvb
+        background-image: linear-gradient(0deg, $accent-color 0%, $accent-color 100%);
+        background-position: bottom left;
+        background-repeat: no-repeat;
+        background-size: 100% 2px;
+        $external-link-selector: '&:not([href^="https://www.datenanfragen.de"]):not([href^="https://www.datarequests.org"]):not([href^="https://www.demandetesdonnees.fr"]):not([href^="https://www.pedidodedados.org"]):not([href^="https://static.dacdn.de"]):not([href^="http://localhost:"]):not([href^="http://datenanfragen.test:"]):not([href^="https://verein.datenanfragen.de"]):not([href^="#"]):not([href*="--datenanfragen.netlify.app/"])';
```

Comment:
Wouldn't `variables.scss` be a better place for that?

Reply:
Why include `localhost` and why only http? For testing? When the changes are live, localhost would be external IMO.

Reply:
> Wouldn't variables.scss be a better place for that?

Imho this is the better place. `variables.scss` is for 'global' variables that are used in multiple places. This doesn't apply here. `$external-link-selector` is only used here and I don't see a case where it would be needed somewhere else.

Reply:
>  Why include localhost and why only http? For testing? When the changes are live, localhost would be external IMO.

The only instance of `localhost` links appearing on the site is when developing locally. In this case, these are indeed internal links. We have always treated `localhost` as such, it also has CORS access for example.

Reply:
* regarding `http`: Because the dev-server unfortunately doesn't work with `https`

Reply:
> Imho this is the better place. `variables.scss` is for 'global' variables that are used in multiple places. This doesn't apply here.

While I agree the selector isn't global, I disagree with the purpose of `variables.scss`. IMO, it is the place for global varibales or all preference-ish variables, so that if you want to tweak the styles to another website, everything you need to change is in there. Context-wise that makes more sense to me. But I don't care much about that.

Reply:
I don't care too much either, so since @mal-tee already changed it, we might as well leave it. *shrug*

Reply:
I feel like, not marking localhost as an external address in production might be a problem in spotting wrong links that appeared correct during local testing. Also, an attacker could use this to place links to localhost on our website that would then appear to be internal. But I guess in that case that would be the least problem. Still, it doesn't feel right to me.

Reply:
I'll let @mal-tee decide, whether they want to change it or not.

Reply:
> I feel like, not marking localhost as an external address in production might be a problem in spotting wrong links that appeared correct during local testing.

I don't see this as a diagnostic tool for use but as an indicator for users on whether a link will lead them to a different website. If you want a tool like that, I wouldn't be opposed to that but wouldn't it be easier to just grep `public` for `localhost` or something similar? That seems way easier and more effective.

 > Also, an attacker could use this to place links to localhost on our website that would then appear to be internal.

I don't really see the attack vector, to be honest. How would they place this link? In that case they would most likely have XSS already anyway, wouldn't they?

Reply:
> I don't really see the attack vector, to be honest. How would they place this link? In that case they would most likely have XSS already anyway, wouldn't they?

Yes, but a link to localhost would maybe allow them to escalate their attack. I just see no reason to leave this avoidable pattern in, other than comfort. It just seems a little sloppy to have dev-only code in production. I know that there is no real risk here and I am fine with merging anyway.

Reply:
I agree with Benjamin, but I can see why you do not want `localhost` in the list. How about we merge this as it is and create a new Issue. Then @zner0L or somebody else can implement it later on.

Reply:
Btw: If it was possible to inject a link, an attacker could inject it in such a way that the selector for this symbol is not applicable.

Reply:
I.e. they could be able to circumvent this indicator even without `localhost` in this list.